onnxconverter_common/float16.py

```diff
@@ -20,7 +20,23 @@ def _npfloat16_to_int(np_list):
     return [int(bin(_.view('H'))[2:].zfill(16), 2) for _ in np_list]
 
 
-def convert_tensor_float_to_float16(tensor):
+def convert_np_to_float16(np_array, min_positive_val=1e-7, max_finite_val=1e4):
```

Comment:
1e-6, or -7? Can you confirm that？

Reply:
![image](https://user-images.githubusercontent.com/67606533/101711265-42176400-3a61-11eb-97b4-3c82c0d5fe3b.png)

Reply:
cool!